librz/main/main.c

```diff
@@ -8,44 +9,35 @@
 
 RZ_LIB_VERSION(rz_main);
 
-static RzMain foo[] = {
-	{ "rz", rz_main_rizin },
+typedef struct main_entry_t {
+	const char *name;
+	RzMainCallback main;
+} MainEntry;
+
+static MainEntry main_prog[] = {
+	{ "rizin", rz_main_rizin },
+	{ "rz-agent", rz_main_rz_agent },
+	{ "rz-asm", rz_main_rz_asm },
 	{ "rz-ax", rz_main_rz_ax },
+	{ "rz-bin", rz_main_rz_bin },
 	{ "rz-diff", rz_main_rz_diff },
-	{ "rz-find", rz_main_rz_find },
-	{ "rz-run", rz_main_rz_run },
-	{ "rz-asm", rz_main_rz_asm },
+	{ "rz-find", rz_main_rz_ax },
 	{ "rz-gg", rz_main_rz_gg },
-	{ "rz-bin", rz_main_rz_bin },
-	{ "rizin", rz_main_rizin },
-	{ NULL, NULL }
+	{ "rz-hash", rz_main_rz_hash },
+	{ "rz-run", rz_main_rz_run },
+	{ "rz-sign", rz_main_rz_sign },
+	{ "rz", rz_main_rizin },
```

Comment:
There might be a tool installed, called `rz` and completely unrelated to the Rizin.